firefox-ios/Client/Application/AppDelegate+PushNotifications.swift

```diff
@@ -65,6 +66,14 @@ extension AppDelegate {
             }
         }
     }
+
+    private func setPreferencesForSyncedAccount(for newState: ConstellationState) {
+        guard self.profile.hasSyncableAccount() else { return }
+        profile.prefs.setBool(true, forKey: PrefsKeys.Sync.signedInFxaAccount)
+        let remoteCount = newState.remoteDevices.count
+        let devicesCount = Int32(remoteCount + 1)
```

Comment:
this extra one is to include the local device as well, let me know if you would like a comment to clarify

Reply:
That would help in future :)

Reply:
added!